src/test/ui/uninhabited/privately-uninhabited-dead-code.rs

```diff
@@ -0,0 +1,17 @@
+// compile-pass
```

Comment:
This test needs a `#![deny(unreachable_code)]` to test the unreachable code lint. compile-pass accepts tests that emit lint warnings but still compile.

Reply:
Oh, good catch!